build.gradle.kts

```diff
@@ -80,7 +86,7 @@ subprojects {
   configure<JavaPluginExtension> {
     toolchain {
       languageVersion.set(JavaLanguageVersion.of(11))
-      vendor.set(JvmVendorSpec.ADOPTOPENJDK)
+      vendor.set(JvmVendorSpec.ADOPTIUM)
```

Comment:
https://docs.gradle.org/7.4-rc-1/release-notes.html

Reply:
Failed on apple silicon:
```log
Execution failed for task ':okhttp:compileKotlinJvm'.
> Error while evaluating property 'kotlinJavaToolchainProvider.javaVersion' of task ':okhttp:compileKotlinJvm'
   > Failed to calculate the value of property 'providedJvm$kotlin_gradle_plugin'.
      > Unable to download toolchain matching these requirements: {languageVersion=11, vendor=ADOPTIUM, implementation=vendor-specific}
         > Unable to download toolchain. This might indicate that the combination (version, architecture, release/early access, ...) for the requested JDK is not available.
            > Could not read 'https://api.adoptopenjdk.net/v3/binary/latest/11/ga/mac/aarch64/jdk/hotspot/normal/adoptopenjdk' as it does not exist.

```

Reply:
https://docs.gradle.org/7.4-rc-1/userguide/toolchains.html#sec:provisioning